jenkins-scripts/tools/outdated-job-runner.groovy

```diff
@@ -0,0 +1,223 @@
+    'ignition_launch-ci-ign-launch2-focal-amd64',
+    'ign_launch-gz-6-win',
+    'ignition_fuel-tools-ci-gz-fuel-tools8-homebrew-amd64',
+    'ignition_utils-ci-gz-utils2-homebrew-amd64',
+    'ignition_math-ci-ign-math6-homebrew-amd64',
+    'ignition_gazebo-ci-gz-sim7-focal-amd64',
+    'ign_fuel-tools-gz-8-win',
+    'ignition_sensors-ci-gz-sensors7-focal-amd64',
+    'sdformat-ci-sdformat12-homebrew-amd64',
+    'ignition_msgs-ci-ign-msgs8-focal-amd64',
+    'ignition_plugin-ci-ign-plugin1-focal-amd64',
+    'ignition_physics-ci-ign-physics2-focal-amd64',
+    'ignition_cmake-ci-gz-cmake3-homebrew-amd64',
+    'ign_plugin-gz-2-win',
+    'ignition_launch-ci-ign-launch5-homebrew-amd64',
+    'ignition_common-ci-gz-common5-focal-amd64',
+    'ign_sensors-ign-3-win',
+    'ign_physics-gz-6-win',
+    'ign_common-ign-3-win',
+    'ign_tools-gz-2-win',
+    'ignition_launch-ci-gz-launch6-homebrew-amd64',
+    'sdformat-sdf-13-win',
+    'ignition_common-ci-ign-common4-homebrew-amd64',
+    'ign_gui-ign-6-win',
+    'ignition_gui-ci-ign-gui6-homebrew-amd64',
+    'ignition_tools-ci-ign-tools1-homebrew-amd64',
+    'ignition_gazebo-ci-ign-gazebo3-focal-amd64',
+    'ignition_cmake-ci-ign-cmake2-focal-amd64',
+    'ign_sensors-gz-7-win',
+    'ignition_fuel-tools-ci-gz-fuel-tools8-focal-amd64',
+    'ignition_rendering-ci-ign-rendering6-homebrew-amd64',
+    'ign_msgs-ign-8-win',
+    'ignition_msgs-ci-gz-msgs9-focal-amd64',
+    'ignition_gazebo-ci-ign-gazebo6-homebrew-amd64',
+    'ign_gui-gz-7-win',
+    'sdformat-sdf-9-win',
+    'ignition_transport-ci-gz-transport12-homebrew-amd64',
+    'ignition_gazebo-ci-ign-gazebo6-focal-amd64',
+    'ignition_launch-ci-gz-launch6-focal-amd64',
+    'ignition_fuel-tools-ci-ign-fuel-tools4-focal-amd64',
+    'ign_cmake-gz-3-win',
+    'ign_gazebo-ign-6-win',
+    'ignition_sensors-ci-ign-sensors3-focal-amd64',
+    'sdformat-ci-sdformat13-homebrew-amd64',
+    'sdformat-ci-sdformat13-focal-amd64',
+    'ignition_common-ci-gz-common5-homebrew-amd64',
+    'ignition_cmake-ci-gz-cmake3-focal-amd64',
+    'ign_transport-ign-11-win',
+    'ign_gazebo-gz-7-win',
+    'ign_sensors-ign-6-win',
+    'ignition_sensors-ci-ign-sensors6-homebrew-amd64',
+    'ignition_gazebo-ci-gz-sim7-homebrew-amd64',
+    'ignition_gui-ci-gz-gui7-homebrew-amd64',
+    'ignition_math-ci-gz-math7-focal-amd64',
+    'ignition_transport-ci-ign-transport11-homebrew-amd64',
+    'ignition_math-ci-gz-math7-homebrew-amd64',
+    'ignition_fuel-tools-ci-ign-fuel-tools7-homebrew-amd64'
+];
+
+def trackedJobs = new LinkedHashSet(trackedJobsList);
+
+def runJob(String jobName) {
+    /*
+     * Schedule a job to run immediately
+     */
+    def job = Jenkins.instance.getItemByFullName(jobName)
+    job.scheduleBuild(0)
+}
+
+def findAvailableNodes() {
+    /*
+     * Find available nodes to run jobs per OS using labels
+     * win: windows
+     * osx: macos
+     * linux: docker
+     */
+    def availableNodes = [osx: [], win: [], linux: []]
```

Comment:
Although we are assuming `docker` equal to `linux` all over the place, I would prefer stop doing that explicitly for the new code. Can we please use `docker` instead of `linux` for the code naming?

Reply:
I renamed all linux->docker in cfa53d7, but I leaved [`linux: docker`](https://github.com/gazebo-tooling/release-tools/pull/962/commits/cfa53d740a53b889245fe2b829966b3b221f05f1#diff-f516195d9cb9e4102140b3a472a627c611d3f1b24a7b99f0388d5c25a9dd0433R23) for clarity purposes